lib/eight_ball.rb

```diff
@@ -85,9 +103,28 @@ def self.with(name, parameters = {})
     yield if enabled? name, parameters
   end
 
+  # Marshalls the {EightBall::Feature Features}. This can be useful for
```

Comment:
Should we consider RDoc or something similar to generate some usage documentation for these things?